metasv/generate_sv_intervals.py

```diff
@@ -93,22 +147,285 @@ def generate_sc_intervals_callback(result, result_list):
     if result is not None:
         result_list.append(result)
 
+def infer_svtype(aln, isize_mean, isize_sd, num_sd=2):
+    min_isize = isize_mean - num_sd * isize_sd
+    max_isize = isize_mean + num_sd * isize_sd
+    if aln.mate_is_unmapped:
+        return "INS"
+    if aln.tid != aln.rnext:
+        return "CTX,INS"
```

Comment:
I fixed the soft-clip length issue (ignoring deleted bases) and substantially revised the generate_final_vcf.py to improve the way we merge duplicate records and filter bad INS calls. Please review these new changes. I also did small test and it seems fine. If changes seems ok you can merge.
I also started a full run which will be done by Thursday night. You may also wait for that to finish.